ANALYSIS/ANALYSISaliceBase/AliAnalysisAlien.cxx

```diff
@@ -1267,7 +1267,7 @@ Bool_t AliAnalysisAlien::CreateDataset(const char *pattern)
    TString path;
    Int_t nruns = 0;
    TString schunk, schunk2;
-   TAliceCollection *cbase=0, *cadd=0, *res=0;
+   TAliceCollection *cbase=0, *cadd=0, *cbase_m=0, *cadd_m=0;
```

Comment:
Hi @Atlantic777,
Would you replace cbase_m and cadd_m by cbaseM and caddM to follow the old coding conventions? We do not allow underscores in the variable or function names...
Thanks,
Peter

Reply:
Hi @pzhristov,
Thanks for the suggestion. I've updated it, the variables are renamed.
Thanks,
Nikola